auth-web/src/components/auth/common/PADInfoForm.vue

```diff
@@ -126,169 +123,155 @@
+    isChangeView: { type: Boolean, default: false },
+    isInitialAcknowledged: { type: Boolean, default: false },
+    isInitialTOSAccepted: { type: Boolean, default: false },
+    isTOSNeeded: { type: Boolean, default: true },
+    padInformation: { default: () => { return {} as PADInfo } }
+  },
+  setup (props, { emit }) {
+    // refs
+    const preAuthDebitForm = ref(null) as HTMLFormElement
+    // store stuff
+    const store = useStore()
+    const currentOrgPADInfo = computed(() => store.state.org.currentOrgPADInfo as PADInfo)
+    const setCurrentOrganizationPADInfo = (padInfo: PADInfo) => store.dispatch('org/updatePadInfo', padInfo)
 
-  private transitNumberRules = [
-    v => !!v || 'Transit Number is required',
-    v => (v.length >= 4) || 'Transit Number should be minimum of 4 digits'
-  ]
+    // static vars
+    const accountMask = CommonUtils.accountMask()
 
-  private institutionNumberRules = [
-    v => !!v || 'Institution Number is required',
-    v => (v.length === 3) || 'Institution Number should be 3 digits'
-  ]
+    const accountNumberRules = [
+      v => !!v || 'Account Number is required',
+      v => (v.length >= 7 && v.length <= 12) || 'Account Number should be between 7 to 12 digits'
+    ]
+    const institutionNumberRules = [
+      v => !!v || 'Institution Number is required',
+      v => (v.length === 3) || 'Institution Number should be 3 digits'
+    ]
+    const transitNumberRules = [
+      v => !!v || 'Transit Number is required',
+      v => (v.length >= 4) || 'Transit Number should be minimum of 4 digits'
+    ]
 
-  private accountNumberRules = [
-    v => !!v || 'Account Number is required',
-    v => (v.length >= 7 && v.length <= 12) || 'Account Number should be between 7 to 12 digits'
-  ]
+    const state = (reactive<PADInfoFormState>({
+      accountNumber: '',
+      bankInfoDialog: false,
+      institutionNumber: '',
+      isAcknowledged: props.isInitialAcknowledged,
+      isTOSAccepted: false,
+      isTouched: false,
+      ready: false,
+      transitNumber: '',
+      showPremiumPADInfo: computed((): boolean => props.isChangeView),
+      acknowledgementLabel: computed((): string => {
+        return (state.showPremiumPADInfo)
+          ? 'I understand that services will continue to be billed to the linked BC Online account until the mandatory (3) day confirmation period has ended.'
+          : 'I understand that this account will not be able to perform any transactions until the mandatory (3) day confirmation period for pre-authorized debit has ended.'
+      }),
+      padInfoSubtitle: computed((): string => {
+        return (state.showPremiumPADInfo)
+          ? 'Services will continue to be billed to the linked BC Online account until the mandatory (3) day confirmation period has ended.'
+          : 'This account will not be able to perform any transactions until the mandatory (3) day confirmation period has ended.'
+      })
+    }) as unknown) as PADInfoFormState
```

Comment:
There might be a better way of doing this? Not sure why you have to cast as unknown

Reply:
I tried a lot of things here. Nothing worked. It works fine without this in ppr / any other composition api I've written but in auth-web the build fails without this

Reply:
I think I'll have another go at this later, but I need to let it simmer / sleep on it a bit before trying again

Reply:
Sounds good, in the mean time.. lets merge it